actions/websocket_actions.jsx

```diff
@@ -291,9 +296,9 @@ function handleEvent(msg) {
     }
 }
 
-function handleChannelUpdatedEvent(msg) {
-    const channel = JSON.parse(msg.data.channel);
-    dispatch({type: ChannelTypes.RECEIVED_CHANNEL, data: channel});
+async function handleChannelUpdatedEvent(msg) {
+    const channelId = msg.data.channel_id;
+    await dispatch(getChannel(channelId));
```

Comment:
You don't need to `await` here if you're not actually using the result

Reply:
Oh right, I've applied this on new function `handleChannelConvertedEvent`. Thanks!